test/snapp_auction.js

```diff
@@ -520,4 +521,31 @@ contract("SnappAuction", async (accounts) => {
       assert.equal(currentAuction.size, 2)
     })
   })
+
+  describe("Larger Test Cases", () => {
+    it("Fill order batch", async () => {
+      const instance = await SnappAuction.new()
+      await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
+      const order = encodeOrder(0, 1, 2, 3)
+      const maxAuctionSize = (await instance.AUCTION_BATCH_SIZE.call()).toNumber()
+      const orders = Array(maxAuctionSize).fill(order)
+      const partitionedOrders = splitArray(orders, 100)
+      let concatenated_orders
+      // let count = 0
+      await partitionedOrders.forEach(function(partition) {  // should be about 10
+        concatenated_orders = Buffer.concat(partition)
+        instance.placeSellOrders(concatenated_orders, { from: user_1 })
+      })
+      // const concatenated_orders = Buffer.concat(orders)
+      // await instance.placeSellOrders(concatenated_orders, { from: user_1 })
+ 
+      const auctionIndex = (await instance.auctionIndex.call()).toNumber()
+      const currentAuction = await instance.auctions(auctionIndex)
+      assert.equal(currentAuction.size.toNumber(), maxAuctionSize)
+ 
+      // The last order should wind up in second batch!
+      await instance.placeSellOrders(order, { from: user_1 })
+      // console.log((await instance.auctionIndex.call()).toNumber())
```

Comment:
missing some assert? Also let's clean up all the commented code that's no longer used.

Reply:
Weird. I thought this was a draft.